helm/ffc-demo-user-service/templates/deployment.yaml

```diff
@@ -13,34 +13,34 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ .Values.name }}
-        environment: {{ .Values.environment }}
+        app: {{ quote .Values.name }}
+        environment: {{ quote .Values.environment }}
       annotations:
-        redeployOnChange: "{{ .Values.container.redeployOnChange }}"
+        redeployOnChange: "{{ quote .Values.container.redeployOnChange }}"
```

Comment:
I think this might be the cause of the build error: we've still got quotes around a value that's quoted using the Helm `quote` function.